common/src/main/kotlin/com/hartwig/actin/molecular/evidence/actionability/ActionabilityMatcher.kt

```diff
@@ -163,21 +167,26 @@ class ActionabilityMatcher(private val evidences: List<EfficacyEvidence>, privat
             emptyList()
         }
 
-        val copyNumberDeletionMatches = if (CopyNumberEvidence.isDeletionEvent(gene.event())) {
-            molecularTest.drivers.copyNumbers
-                .filter { copyNumber ->
-                    CopyNumberEvidence.isDeletionMatch(gene, copyNumber)
-                }
-        } else {
-            emptyList()
-        }
+        val copyNumberDeletionMatches =
+            if (CopyNumberEvidence.isDeletionEvent(gene.event()) || isMmrAbsenceOfProteinEvent(gene.event(), gene.gene())) {
+                molecularTest.drivers.copyNumbers
+                    .filter { copyNumber ->
+                        CopyNumberEvidence.isDeletionMatch(gene, copyNumber)
+                    }
+            } else {
+                emptyList()
+            }
 
         return successWhenNotEmpty(
             variantMatches + promiscuousFusionMatches + disruptionMatches +
                     homozygousDisruptionMatches + copyNumberAmplificationMatches + copyNumberDeletionMatches
         )
     }
 
+    private fun isMmrAbsenceOfProteinEvent(geneEvent: GeneEvent, gene: String): Boolean {
+        return geneEvent == GeneEvent.ABSENCE_OF_PROTEIN && GeneConstants.MMR_GENES.contains(gene)
+    }
```

Comment:
I would encapsulate this into existing functions, e.g. `VariantEvidence.isGeneEventEligible` should include this rather than be a separate function